langlib/lang.xml/src/main/java/org/ballerinalang/langlib/xml/ElementChildren.java

```diff
@@ -29,35 +29,54 @@
 import org.ballerinalang.natives.annotations.ReturnType;
 
 import java.util.ArrayList;
+import java.util.List;
 
 /**
  * Return lift getChildren over sequences.
  *
  * @since 1.2
  */
 @BallerinaFunction(
-        orgName = "ballerina", packageName = "lang.__internal",
-        functionName = "children",
-        args = {@Argument(name = "xmlValue", type = TypeKind.XML)},
+        orgName = "ballerina", packageName = "lang.xml",
+        functionName = "elementChildren",
+        args = {@Argument(name = "xmlValue", type = TypeKind.XML), @Argument(name = "name", type = TypeKind.STRING)},
```

Comment:
Shall we use the same param names as in .bal file
And also type of parameter `name` is not String it's a union of string and nil